graphql/fetchAllUserAssignments.ts

```diff
@@ -0,0 +1,19 @@
+import { gql } from "@apollo/client";
+export const FETCH_ALL_USER_ASSIGNMENTS = gql`
```

Comment:
let's rewrite this query so we get assignment name from the coding_assignments table - this should be possible since both tables have a relationship defined